unittests/tools/test_cyberwatch_parser.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from pathlib import Path
+
+from dojo.models import Test
+from dojo.tools.cyberwatch.parser import CyberwatchParser
+
+
+class TestCyberwatchParser(unittest.TestCase):
+
+    def setUp(self):
+        self.parser = CyberwatchParser()
+        self.test = Test()
+
+    def test_no_findings(self):
+        testfile = Path("unittests/scans/cyberwatch/no_findings.json")
+        with testfile.open("rb") as file:
+            findings = self.parser.get_findings(file, self.test)
+            self.assertEqual(0, len(findings))
+
+    def test_one_security_issue(self):
+        testfile = Path("unittests/scans/cyberwatch/one_security_issue.json")
+        with testfile.open("rb") as file:
+            findings = self.parser.get_findings(file, self.test)
+            self.assertEqual(1, len(findings))
+
+            finding = findings[0]
+            self.assertEqual("Security Issue - Fingerprint Web Application Framework", finding.title)
+            self.assertEqual("Info", finding.severity)
+            # Expect both endpoints to have the same host as per new JSON
+            endpoint_hosts = [e.host for e in finding.unsaved_endpoints]
```

Comment:
Could you add a `.clean()` for all endpoints? This validates if they are in valid format.